src/pages/hcp/api-docs/vault-secrets/[[...page]].tsx

```diff
@@ -51,6 +54,28 @@ const PRODUCT_SLUG = 'hcp'
  */
 const BASE_URL = '/hcp/api-docs/vault-secrets'
 
+/**
+ * Resource items are shown in the sidebar
+ */
+const NAV_RESOURCE_ITEMS: OpenApiNavItem[] = [
```

Comment:
💡 I wonder if a `constants` file might be helpful for all these config values.

Reply:
ooh i like that a lot! will add in a follow-up PR 👍